styles.css

```diff
@@ -1,399 +1,692 @@
+  background-position: 0% -10%;
+  background-size: 100px;
+}
+
+.shining-star-11 {
+  text-align: center;
+  width: 50%;
+  margin: 0 auto;
+}
+
+.shining-star-2 {
+  background-image: url(bg/eb.png);
+  background-repeat: no-repeat;
+  background-position: 100% 5%;
+  background-size: 100px;
+  margin-right: -30px;
+}
+
+.Technology-knowledge {
+  border-top: 1px solid #646369;
+  padding-top: 20px;
+  display: flex;
+  flex-wrap: wrap;
+  padding-left: 10px;
+  padding-bottom: 20px;
+  border-bottom: 1px solid #646369;
+}
+
+.Technology-knowledge h4 {
+  padding-bottom: 20%;
+  text-align: left;
+  font-family: 'Roboto', sans-serif;
+  font-weight: 500;
+}
+
+.Technology-knowledge li {
+  font-weight: 400;
+  padding: 6px 0;
+  text-align: left;
+  font-family: 'Roboto', sans-serif;
+  width: 50%;
+  line-height: 24px;
+  font-size: 15px;
+}
+
+.heading {
+  color: #9490ff;
+}
+
+.headinglist {
+  color: #edecff;
+}
+
+.icon-list {
+  display: flex;
+  justify-content: center;
+  gap: 0;
+}
+
+.lang-list {
+  display: flex;
+  justify-content: space-between;
+  padding-bottom: 20px;
+  gap: 0;
+}
+
+.nav-arrow {
+  display: flex;
+  justify-content: center;
+}
+
+.arrow {
+  margin: 6px;
+  width: 28px;
+  height: 20px;
+  filter: opacity(1) drop-shadow(0 0 0 #a2a3ae) saturate(300%) contrast(0%) blur(0);
+}
+
+.navarrow {
+  display: none;
+}
+
+.recent-work-heading {
+  font-size: 20px;
+  line-height: 32px;
+  color: #a2a3ae;
+}
+
+.works {
+  text-decoration: none;
+}
 
+.works > p {
+  font-size: 20px;
+  line-height: 32px;
+  color: #a2a3ae;
+}
+
+.works:hover {
+  cursor: pointer;
+  text-decoration: 2px underline var(--secondary-heading-color);
+}
+
+.header-language {
+  padding: 0 30px 60px 30px;
+}
+
+.recent-work-heading {
+  font-size: 32px;
+  margin-top: 20px;
+  font-family: 'Roboto', sans-serif;
+  color: #9490ff;
+  width: 220px;
+  text-transform: uppercase;
+}
+
+.grid-container {
+  background-image: url(bg/ec.png), url(bg/ed.png), url(bg/ee.png), url(bg/ef.png);
+  background-repeat: no-repeat, no-repeat, no-repeat, no-repeat;
+  background-position: left top 7%, right top 36%, left top 63%, right bottom 8%;
+  background-size: 250px;
+  display: grid;
+  grid-template-rows: auto;
+  padding: 0 30px;
+  gap: 10px;
+  width: 100%;
+}
+
+.grid-item {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 20px;
+}
+
+.grid-item p {
+  color: #dcdce8;
+  font-size: 20px;
+  line-height: 28px;
+  font-weight: 400;
+  font-family: 'Roboto', sans-serif;
+  padding-bottom: 10px;
+}
+
+.project-title {
+  font-size: 32px;
+  font-weight: 500;
+  line-height: 44px;
+  margin: 50px 0;
+  font-family: 'Roboto', sans-serif;
+  color: white;
+}
+
+.btn {
+  height: 48px;
+  width: 89px;
+  background-color: #7d73fc;
+  color: white;
+  padding: 12px;
+  align-items: center;
+  font-size: 12px;
+  border: none;
+  font-family: 'Roboto', sans-serif;
+  line-height: 24px;
+  font-weight: 500px;
+  letter-spacing: 3%;
+}
```

Comment:
- [ ] [OPTIONAL] It would be better if you implement different button styles based on user interaction like when the **user hovers** on it or **presses it.** These enhance your design and also increase user experience while interacting with your website 💡. They have provided design guidelines in the Figma design you chose so you can refer to them while implementing them. 👇 
![image](https://user-images.githubusercontent.com/90222110/179231733-347ddd6f-8e9d-4988-a5d4-bea2698767f5.png)